i/inputplumber/stone.yaml

```diff
@@ -0,0 +1,34 @@
+#
+# SPDX-FileCopyrightText: © 2020-2024 Serpent OS Developers
+#
+# SPDX-License-Identifier: MPL-2.0
+#
+name: inputplumber
+version: v0.40.0
+release: 1
+homepage: https://github.com/ShadowBlip/InputPlumber
+upstreams:
+  - https://github.com/ShadowBlip/InputPlumber/archive/refs/tags/v0.40.0.tar.gz: b156bb188e3519e835f04793f1e11a9262e3014c8e8bdc61786f41c24f93f7d9
+summary: Open source input router and remapper daemon for Linux
+description: |
+  InputPlumber is an open source input routing and control daemon for Linux.
+  It can be used to combine any number of input devices (like gamepads, mice, and keyboards)
+  and translate their input to a variety of virtual device formats.
+networking: yes
+builddeps:
+  - binary(cargo)
+  - binary(clang)
+  - binary(llvm-config)
+  - clang-devel
```

Comment:
Are these dependencies actually used? `binary(clang)` is unnecessary as it's the default toolchain on Serpent and is pulled in automatically, and I don't see that this is linking against libclang or libllvm which I'd expect for a clang-devel/llvm-config dependency.

Reply:
Without `binary(llvm-config)` it doesn't find llvm-config and without `clang-devel` the libclang.so is not found. I dropped binary clang.